### PR TITLE
Fix CMakeLists install dir & Warnings

### PR DESCRIPTION
**Pull request by swetha097**

### CODE REVIEW

File: utilities/rocAL/rocAL_basic_test/CMakeLists.txt [hunk 1]
```diff
@@ -35,7 +35,7 @@ list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../amd_openvx/cmake)
 find_package(OpenCV QUIET)
 find_package(AMDRPP QUIET)
 
-include_directories(${ROCM_PATH}/libexec/mivisionx/rali/python/)
```
Comment (kiritigowda) [inline utilities/rocAL/rocAL_basic_test/CMakeLists.txt:38]: scripts need to install at -- ${CMAKE_INSTALL_LIBEXECDIR}/mivisionx/rocAL

File: rocAL/rocAL/CMakeLists.txt [hunk 2]
```diff
@@ -144,7 +144,7 @@ if(${BUILD_ROCAL})
         include/api/rocal_api_parameters.h
         include/api/rocal_api_meta_data.h
         # DESTINATION include/mivisionx
-        DESTINATION ${CMAKE_INSTALL_LIBEXECDIR}/mivisionx/rali/python
```
[the remark this answers is not in the document]
Comment (LakshmiKumar23) [reply rocAL/rocAL/CMakeLists.txt:147]: @swetha097 Please look into these comments from Kiriti

[the remark this answers is not in the document]
Comment (swetha097) [reply rocAL/rocAL/CMakeLists.txt:147]: Done

File: utilities/rocAL/rocAL_basic_test/CMakeLists.txt (hunk 1 above)
Comment (swetha097) [reply utilities/rocAL/rocAL_basic_test/CMakeLists.txt:38]: Done

**Comment by kiritigowda** [comment]
@swetha097 can you fix the merge conflicts

File: utilities/rocAL/rocAL_dataloader/rocAL_dataloader.cpp [hunk 3]
```diff
@@ -165,7 +165,7 @@ int main(int argc, const char ** argv)
      RocalImage image0;
      image0 = input1;
     // just do one augmentation to test
-    // image1 = rocalRain(handle, image0, true);
+    rocalRain(handle, image0, true);
```
Comment (LakshmiKumar23) [inline utilities/rocAL/rocAL_dataloader/rocAL_dataloader.cpp:168]: @swetha097 Was this change meant to be in this PR?

File: rocAL/rocAL/source/pipeline/master_graph.cpp [hunk 4]
```diff
@@ -111,7 +111,7 @@ MasterGraph::~MasterGraph()
     release();
 }
 
-MasterGraph::MasterGraph(size_t batch_size, RocalAffinity affinity, int gpu_id, size_t cpu_threads, size_t prefetch_queue_depth, RocalTensorDataType output_tensor_data_type):
+MasterGraph::MasterGraph(size_t batch_size, RocalAffinity affinity, int gpu_id, size_t prefetch_queue_depth, RocalTensorDataType output_tensor_data_type):
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/source/pipeline/master_graph.cpp:114]: @swetha097 This also?

Comment (swetha097) [reply rocAL/rocAL/source/pipeline/master_graph.cpp:114]: This resolved some of the warnings Lakshmi

**Comment by LakshmiKumar23** [comment]
@swetha097 I don't see the CmakeLists changes it started with

File: utilities/rocAL/rocAL_dataloader/rocAL_dataloader.cpp (hunk 3 above)
Comment (swetha097) [reply utilities/rocAL/rocAL_dataloader/rocAL_dataloader.cpp:168]: Warnings were also resolved in addition the CMakeLists changes

**Review by LakshmiKumar23 — APPROVED** [review]

**Comment by swetha097** [comment]
> @swetha097 I don't see the CmakeLists changes it started with

I just merged with TOT, looks like it was fixed in TOT

**Comment by LakshmiKumar23** [comment]
> > @swetha097 I don't see the CmakeLists changes it started with
> 
> I just merged with TOT, looks like it was fixed in TOT

@swetha097 @kiritigowda Yeah I checked too, it's in TOT. Approved the PR. Can merge once CI passes. I just started CI

**Comment by LakshmiKumar23** [comment]
@kiritigowda can be merged. [CI passes](http://math-ci.rocm.amd.com/blue/organizations/jenkins/compute-rocm-dkms-no-npi-hipclang%2FShort-GPUOpen%2FMIVisionX/detail/PR-896/5/pipeline/2083/). Not sure why it shows otherwise.
@swetha097 Still a lot of warnings for the rocAL parts. We can fix it in another PR

**Pull request merged**